from MIWOS.libs.exceptions.MIWOS_exception import MIWOSException


class ValidationException(MIWOSException):
    pass
